src/app/list/list.component.scss

```diff
@@ -0,0 +1,69 @@
+.app-list-container{
+    width: 100%;
+    height: 100%;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+}
+
+.app-list{
+    width: 700px;
+    min-height: 400px;
+    height: fit-content;
+    
+}
+
+.app-list-header{
+    width:100%;
+    height: 50px;    
+    // background-image: linear-gradient(90deg, rgba(161,79,215,1) 0%, rgba(253,29,29,1) 50%, rgba(252,176,69,1) 100%);
+    border: 1px solid rgba(0,0,0,0.4);
+    background: rgba(0,0,0,0.1);
+    border-radius: 6px 6px 0px 0px;
+    color: rgba(0,0,0,0.8);
+    font-size: 1.5rem;
+
+    display: flex;
+    align-items: center;
+    justify-content: center;
+}
+
+.app-list-header>*{
+    height: 100%;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+}
+
+
+
+.app-list-content{
+    width: 100%;
+    margin:0;
+    padding:0;
+}
+
+.app-list-content-element{
+    box-sizing: border-box;
+    width: 100%;
+    height: 50px;
+    border: 1px solid rgba(0,0,0,0.4);;
+    color:rgba(0,0,0,0.8);
+    list-style: none;
+    display: flex;
+    align-items: center;
+    justify-content: center;
+}
+
+.app-list-content-element:hover{
+    background-color: rgba(252, 176, 69, 0.4);
+    cursor: pointer;
+}
+
+.app-list-content-element>*{
+    height: 100%;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    font-size: 1.25rem;
```

Comment:
in scss you can make a nicer syntax using &  i.e. 
.app-list-content-element{
  &:hover {
  
  }
  
  &>* {
  
  }
}